master/inventory.py

```diff
@@ -151,15 +154,17 @@ def build_names(platform, arch, name):
         tar_arch = 'aarch64'
         up_arch = 'aarch64'
         bits = 'aarch64'
-        march = 'armv8-a'
+        mcpu = 'armv8-a'    # Better to set mcpu on arm https://github.com/JuliaCI/julia-buildbot/issues/148
         # We have a lot of cores and a lot of RAM, use them.
         nthreads = 8
         flags += 'JULIA_CPU_TARGET=generic '
 
     # Add MARCH to flags
     if not march is None:
         flags += "MARCH=%s "%(march)
-
+    # Add MCPU to flags
+    if not mcpu is None:
+        flags += "MCPU=%s "%(mcpu)
```

Comment:
I don't think the Julia buildsystem uses `MCPU`, does it?  We [have `MARCH` defined](https://github.com/JuliaLang/julia/blob/master/Make.inc#L671), but there's no hint of `MCPU` anywhere.  I think you'd need to plumb it through down here to be added to the compiler invocations: https://github.com/JuliaLang/julia/blob/86ee57cdd77d402ea63108c16df7117d0ce83dc7/Make.inc#L820-L822

Reply:
Yeah, sorry forgot to link the PR I did here https://github.com/JuliaLang/julia/pull/35617